fastlane/Fastfile

```diff
@@ -5,7 +5,7 @@ files_with_version_number = {
     './android/src/main/java/com/revenuecat/purchases/capacitor/PurchasesPlugin.kt' => ['PLUGIN_VERSION = "{x}"']
 }
 files_to_update_phc_version = {
-    'RevenueCatPurchasesCapacitor.podspec' => ['"PurchasesHybridCommon", \'{x}\''],
+    'RevenueCatPurchasesCapacitor.podspec' => ["'PurchasesHybridCommon', \'{x}\'"],
```

Comment:
Do you need to add `\` before the `'`? I don't think you do so you can probably just do

```suggestion
    'RevenueCatPurchasesCapacitor.podspec' => ["'PurchasesHybridCommon', '{x}'"],
```

Reply:
You're right! Good catch!